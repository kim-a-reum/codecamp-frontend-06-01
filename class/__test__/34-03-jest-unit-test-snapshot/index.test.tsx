import { render } from "@testing-library/react"
import JestUnitTestSnapShotPage from "../../pages/34-03-jest-unit-test-snapshot"

it("컴포넌트가 기존이랑 바뀐게 없는지 비교해보자 - 스냅샷 테슷흐",()=>{     
    const result = render(<JestUnitTestSnapShotPage/>)
    expect(result.container).toMatchSnapshot()

})