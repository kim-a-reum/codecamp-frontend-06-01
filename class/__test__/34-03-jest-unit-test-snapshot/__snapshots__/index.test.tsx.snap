// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`컴포넌트가 기존이랑 바뀐게 없는지 비교해보자 - 스냅샷 테슷흐 1`] = `
<div>
  <div>
    아름이는 13살 입니다.
  </div>
  아름이의 취미 입력하기 : 
  <input
    type="text"
  />
  <button>
    아름이랑 놀러가기
  </button>
</div>
`;
